backend/src/cells.rs

```diff
@@ -126,11 +126,11 @@ fn floodfill(map: &MapModel, start: RoadID, neighbourhood: &Neighbourhood) -> Ce
 
             for next in &map.get_i(i).roads {
                 let next_road = map.get_r(*next);
-                /*if let Some(ref filter) = map.get_i(i).modal_filter {
```

Comment:
The code to do this was already here, probably backported from v1. I thought about using `allowed_movements_from` here as a larger cleanup, but it's a bigger change to do the floodfill by intersection, not by road. Maybe when we try #324, this could be useful